### ISSUE TITLE
Search query don't shows existing itmens

### ISSUE THREAD

**Issue by Gmork82** [seek4science/seek#471]
In the results of a search query with a specific keyword, the result is missing if the keyword is in brackets or part of a hyphen separated combination.

1. click on the Search form

2. type in _IMISE_

Expectation: Listed Item of the Institute
https://fairdomhub.org/institutions/269

**Comment by jonovik** [comment]
I think I'm seeing the same.

Searching https://fairdomhub.org/projects/38/search?q=mc6 fails to return [MC6: ...](https://fairdomhub.org/studies/1058) (Study, not public). From what @ReneHaensel says, this could be the colon's fault.

**Comment by stuzart** [comment]
Yes, it is because of the punctuation and is something that needs improving. We did once use a tokenizer that ignored punctuation like this, and would have solved your problems, but we then got complaints about missing results for queries that depends on them, such as certain identifiers or chemical names. It needs fixing by using a combination of tokenizers that indexes both cases.
We do have plans in the coming months to have some dedicated focus on search, and explore different tokenizers and query analysers that will allow more sophisticated searches.

There is a related issue to do with umlauts and accented characters which can be tackled at the same time #1148
